tests/crypto/rand32/src/main.c

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+#include <ztest.h>
+
+extern void test_rand32(void);
```

Comment:
can you please move the main test from rand32.c into this, no need to have 2 files here for such a simple test.